src/Parser/Mage/Shared/Modules/Features/RuneOfPower.js

```diff
@@ -53,32 +61,36 @@ class RuneOfPower extends Analyzer {
     };
   }
 
+
+  get roundedSecondsSuggestionThresholds() {
+    return {
+      actual: this.damageIncreasePercent,
+      isLessThan: {
+        minor: RUNE_DURATION,
+        average: RUNE_DURATION - 1,
+        major: RUNE_DURATION - 3,
+      },
+      style: 'percentage',
+    };
+  }
+
   suggestions(when) {
-    when(this.damageIncreasePercent).isLessThan(this.damageSuggestionThresholds.isLessThan.minor)
+    when(this.damageSuggestionThresholds)
       .addSuggestion((suggest, actual, recommended) => {
-        return suggest(<span>Your <SpellLink id={SPELLS.RUNE_OF_POWER_TALENT.id} /> damage boost is below the expected passive gain from <SpellLink id={SPELLS.INCANTERS_FLOW_TALENT.id} />. Either find ways to make better use of the talent, or switch to <SpellLink id={SPELLS.INCANTERS_FLOW_TALENT.id} />.</span>)
+        return suggest(<Wrapper>Your <SpellLink id={SPELLS.RUNE_OF_POWER_TALENT.id} /> damage boost is below the expected passive gain from <SpellLink id={SPELLS.INCANTERS_FLOW_TALENT.id} />. Either find ways to make better use of the talent, or switch to <SpellLink id={SPELLS.INCANTERS_FLOW_TALENT.id} />.</Wrapper>)
           .icon(SPELLS.RUNE_OF_POWER_TALENT.icon)
           .actual(`${formatPercentage(this.damageIncreasePercent)}% damage increase from Rune of Power`)
-          .recommended(`${formatPercentage(recommended)}% is the passive gain from Incanter's Flow`)
-          .regular(this.damageSuggestionThresholds.isLessThan.average).major(this.damageSuggestionThresholds.isLessThan.major);
+          .recommended(`${formatPercentage(recommended)}% is the passive gain from Incanter's Flow`);
       });
-
-    const casts = this.abilityTracker.getAbility(SPELLS.RUNE_OF_POWER_TALENT.id).casts;
-    if (!casts) {
-      return;
+    if (this.abilityTracker.getAbility(SPELLS.RUNE_OF_POWER_BUFF.id).casts < 1) {
+      when(this.roundedSecondsSuggestionThresholds)
+        .addSuggestion((suggest, actual, recommended) => {
```

Comment:
This should be ``this.abilityTracker.getAbility(SPELLS.RUNE_OF_POWER_BUFF.id).casts > 0`` I guess?